uploadNeuroDB/minc_insertion.pl

```diff
@@ -430,7 +430,8 @@
    ) {
    $message = "\n  --> The minc file cannot be registered ".
                 "since $acquisitionProtocol ".
-                "does not exist in $profile \n";
+                "does not exist in $profile ". 
+                "or it did not pass the extra_file_checks\n";
```

Comment:
@gluneau 

looks good to me. This puts it in line with the logic in the function registerScanIntoDB().

It would have been better to perhaps split the message as follows:
if ($checks->[0] !~ /exclude/) {keep the original message before your changes}
else { add the new part of your message}

But in all cases, your change is necessary as the initial message was misleading when the acquisition was in the prod file but extra_file_checks failed.